org.eclipse.wildwebdeveloper.feature/feature.xml

```diff
@@ -2,7 +2,7 @@
 <feature
       id="org.eclipse.wildwebdeveloper.feature"
       label="%name"
-      version="0.7.2.qualifier"
+      version="0.7.3.qualifier"
```

Comment:
Given that version 0.8.0 was just officially released, why are there older version numbers here?

Reply:
@mickaelistria would have to say that :)

Reply:
I'm going to merge this PR as it's semver related. E.g. version 0.8 is due to the API/extension point added in the xml bundle. I still think that feature version should reflect that though.